requirements.txt

```diff
@@ -3,5 +3,5 @@ Flask>=0.12
 oauthlib~=3.0
 qrcode~=7.3.1
 pillow~=9.4.0
-ovos-utils~=0.0, >=0.0.26
+ovos-utils>=0.0.26,<0.2.0
```

Comment:
The existing spec should allow 0.1.0 too; it is equivalent to `>=0.0.26,<1.0`

Reply:
🤦🏼